container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -33,14 +33,14 @@ const (
 	// in case it needs to be configured by the user down the line
 	k8sApiServerUrl = "https://kubernetes.default.svc:443"
 
-	// TODO: construct fluentbit config via go templating based on inputs
 	fluentBitConfigFileName = "fluent-bit.conf"
 	fluentBitConfigTemplate = `
 [SERVICE]
     HTTP_Server       On
     HTTP_Listen       0.0.0.0
     HTTP_PORT         {{ .HTTPPort }}
     Parsers_File      /fluent-bit/etc/parsers.conf
+    Parsers_File      {{ .KurtosisParsersConfigFilepath }}
```

Comment:
Looks like this is the same template file from Docker. Should this be extracted into a common location, and then fed to both Docker & k8s (presuming that they should be using the same version of Fluentbit, which seems sensible given there's common `parsers`/`filters` elements in the Kurtosis config  being used for both)

Reply:
at some point they should - but wil leave that refactor for a separate PR